client/lib/importer/test/api-payloads/defunct-importer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "importId": "defunctImporter",
+  "type": "wordpress",
+  "importStatus": "importStopped",
+  "siteId": 0
+}
```

Comment:
Nitpicking: This and the file below don't have a end-of-file newline. This is really only specified in the [JavaScript guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#spacing) though I don't see why it shouldn't apply to all files. Our [`.editorconfig` specifies it](https://github.com/Automattic/wp-calypso/blob/52caa3bd5e5f36c829e2807a8894165d22b6d9af/.editorconfig#L9), and is easy to manage if using an [EditorConfig plugin](http://editorconfig.org/).